frontend/encore/bootstrap.rst

```diff
@@ -1,18 +1,24 @@
-Using Bootstrap CSS & JS
-========================
+Using Bootstrap 4 CSS & JS
+============================
 
 Want to use Bootstrap (or something similar) in your project? No problem!
 First, install it. To be able to customize things further, we'll install
-``bootstrap-sass``:
+``bootstrap`` and popper.js:
 
 .. code-block:: terminal
 
-    $ yarn add bootstrap-sass --dev
+    $ yarn add bootstrap popper.js --dev
 
-Importing Bootstrap Sass
+or if you prefer to use npm:
```

Comment:
We decided to use `yarn` in the Symfony Docs. This is of course a decision that can be debated and changed. But meanwhile we can't change individual articles. So, let's remove this change about "npm" and the other one later in this article.